apps/game/src/sandbox/dashboards/trip_table.rs

```diff
@@ -698,3 +715,37 @@ fn make_table_unfinished_trips(app: &App) -> Table<App, UnfinishedTrip, Filters>
 
     table
 }
+
+
+fn export_trip_table(app: &App) -> Result<String, anyhow::Error> {
+    let (finished, _) = produce_raw_data(app);
+    let path = format!(
+        "trip_table_{}_{}.csv",
+        app.primary.map.get_name().as_filename(),
+        app.primary.sim.time().as_filename()
+    );
+    
+    let mut out = std::io::Cursor::new(Vec::new());
+    // Write header
+    writeln!(
+        out,
+        "id,mode,modified,departure,duration,waiting_time,percent_waiting,duration_before"
+    )?;
+    
+    for trip in finished {
+        writeln!(
+            out,
+            "{},{:?},{},{},{},{},{},{}",
+            trip.id.0,
+            trip.mode,
+            trip.modified,
+            trip.departure,
+            trip.duration_after.inner_seconds(),
+            trip.waiting.inner_seconds(),
+            trip.percent_waiting,
+            trip.duration_before.inner_seconds()
+        )?;
+    }
+    
+    abstio::write_file(path, String::from_utf8(out.into_inner())?).map_err(anyhow::Error::from)
```

Comment:
You can leave off the `map_err`, because `write_file` already returns an `anyhow::Result`